From 03f74167ee37420164d22c081ce07d90b4da01a9 Mon Sep 17 00:00:00 2001
From: Paolo Pisati <p.pisati@canonical.com>
Date: Fri, 28 Jul 2023 13:22:46 +0000
Subject: [PATCH] linux 6.5 buildfix

Signed-off-by: Paolo Pisati <p.pisati@canonical.com>
---
 src/r8168_n.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/r8168_n.c b/src/r8168_n.c
index 7e9e2ea..8716c69 100755
--- a/src/r8168_n.c
+++ b/src/r8168_n.c
@@ -81,6 +81,10 @@
 #include <linux/mdio.h>
 #endif
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,5,0)
+#include <net/gso.h>
+#endif
+
 #include <asm/io.h>
 #include <asm/irq.h>
 
-- 
2.40.1

